src/charm.py

```diff
@@ -58,8 +58,10 @@ def __init__(self, *args):
         self._remote_write = PrometheusRemoteWriteConsumer(self)
         self._scrape = MetricsEndpointConsumer(self)
 
-        self._loki_consumer = LokiPushApiConsumer(self)
-        self._log_proxy = LogProxyProvider(self)
+        self._loki_consumer = LokiPushApiConsumer(self, relation_name="logging-consumer")
```

Comment:
Rename to `LokiPushApiRequirer` throughout

Reply:
As discussed today we are holding off on this for consistency.